src/components/Progress.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { CircularProgress } from "@mui/material";
+import customColors from "assets/customColors";
+
+const Progress = ({ progress }) => {
```

Comment:
These progress bars are looking awesome!

There's a slight overflow of the progress bars at smaller screens. Is this something you want to fix? 
![image](https://github.com/Code-the-Dream-School/ee-prac-team3-front/assets/6400323/4432eae6-632d-40c8-9d2f-1cc09432dfb8)